pkg/util/defaults/defaults.go

```diff
@@ -29,13 +29,13 @@ const (
 	DefaultRuntimeVersion = "1.13.0"
 
 	// BuildahVersion --
-	BuildahVersion = "1.14.0"
+	BuildahVersion = "1.23.3"
 
 	// KanikoVersion --
 	KanikoVersion = "0.17.1"
 
 	// baseImage --
-	baseImage = "adoptopenjdk/openjdk11:slim"
+	baseImage = "docker.io/adoptopenjdk/openjdk11:slim"
```

Comment:
Is this needed by the new Buildah?

Reply:
No, i agree with you so i proceed with revert this modify